planning/behavior_velocity_planner/test/src/test_behavior_velocity_planner_node_interface.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ament_index_cpp/get_package_share_directory.hpp"
+#include "behavior_velocity_planner/node.hpp"
+#include "planning_interface_test_manager/planning_interface_test_manager.hpp"
+#include "planning_interface_test_manager/planning_interface_test_manager_utils.hpp"
+
+#include <gtest/gtest.h>
+
+#include <cmath>
+#include <vector>
+
+TEST(PlanningModuleInterfaceTest, NodeTestWithExceptionPathWithLaneID)
+{
+  rclcpp::init(0, nullptr);
+
+  auto test_manager = std::make_shared<planning_test_utils::PlanningInterfaceTestManager>();
+
+  auto node_options = rclcpp::NodeOptions{};
+
+  const auto planning_test_utils_dir =
+    ament_index_cpp::get_package_share_directory("planning_test_utils");
+  const auto behavior_velocity_planner_dir =
+    ament_index_cpp::get_package_share_directory("behavior_velocity_planner");
+  const auto motion_velocity_smoother_dir =
+    ament_index_cpp::get_package_share_directory("motion_velocity_smoother");
+
+  test_utils::updateNodeOptions(
+    node_options,
+    {planning_test_utils_dir + "/config/test_common.param.yaml",
+     planning_test_utils_dir + "/config/test_nearest_search.param.yaml",
+     planning_test_utils_dir + "/config/test_vehicle_info.param.yaml",
+     motion_velocity_smoother_dir + "/config/default_motion_velocity_smoother.param.yaml",
+     motion_velocity_smoother_dir + "/config/Analytical.param.yaml",
+     behavior_velocity_planner_dir + "/config/behavior_velocity_planner.param.yaml",
+     behavior_velocity_planner_dir + "/config/blind_spot.param.yaml",
+     behavior_velocity_planner_dir + "/config/crosswalk.param.yaml",
+     behavior_velocity_planner_dir + "/config/detection_area.param.yaml",
+     behavior_velocity_planner_dir + "/config/intersection.param.yaml",
+     behavior_velocity_planner_dir + "/config/no_stopping_area.param.yaml",
+     behavior_velocity_planner_dir + "/config/occlusion_spot.param.yaml",
+     behavior_velocity_planner_dir + "/config/run_out.param.yaml",
+     behavior_velocity_planner_dir + "/config/speed_bump.param.yaml",
+     behavior_velocity_planner_dir + "/config/stop_line.param.yaml",
+     behavior_velocity_planner_dir + "/config/traffic_light.param.yaml",
+     behavior_velocity_planner_dir + "/config/virtual_traffic_light.param.yaml",
+     behavior_velocity_planner_dir + "/config/out_of_lane.param.yaml"});
+
```

Comment:
Please set the launch flag directly in the test. These parameters must be `true` for all, but it fails for now. I just copied from config.

```suggestion

  node_options.append_parameter_override("launch_stop_line", true);
  node_options.append_parameter_override("launch_crosswalk", true);
  node_options.append_parameter_override("launch_traffic_light", true);
  node_options.append_parameter_override("launch_intersection", true);
  node_options.append_parameter_override("launch_blind_spot", true);
  node_options.append_parameter_override("launch_detection_area", true);
  node_options.append_parameter_override(
    "launch_virtual_traffic_light", false);                                // TODO() set to true
  node_options.append_parameter_override("launch_occlusion_spot", false);  // TODO() set to true
  node_options.append_parameter_override("launch_no_stopping_area", true);
  node_options.append_parameter_override("launch_run_out", false);     // TODO() set to true
  node_options.append_parameter_override("launch_speed_bump", false);  // TODO() set to true
  node_options.append_parameter_override("launch_out_of_lane", true);

```

Reply:
I set flag as following in `generateNode()` function
```
  node_options.append_parameter_override("launch_stop_line", true);
  node_options.append_parameter_override("launch_crosswalk", true);
  node_options.append_parameter_override("launch_traffic_light", true);
  node_options.append_parameter_override("launch_intersection", true);
  node_options.append_parameter_override("launch_blind_spot", true);
  node_options.append_parameter_override("launch_detection_area", true);
  node_options.append_parameter_override(
    "launch_virtual_traffic_light", false);                                // TODO() set to true
  node_options.append_parameter_override("launch_occlusion_spot", false);  // TODO() set to true
  node_options.append_parameter_override("launch_no_stopping_area", true);
  node_options.append_parameter_override("launch_run_out", false);     // TODO() set to true
  node_options.append_parameter_override("launch_speed_bump", false);  // TODO() set to true
  node_options.append_parameter_override("launch_out_of_lane", true);
```